extensions/Worker.Extensions.Http.AspNetCore/src/Worker.Extensions.Http.AspNetCore.csproj

```diff
@@ -1,23 +1,54 @@
-    <Description>ASP.NET Core extensions for .NET isolated functions</Description>
-
-    <!--Version information-->
-    <VersionPrefix>1.0.0</VersionPrefix>
-    <TargetFramework>net6.0</TargetFramework>
-  </PropertyGroup>
-
-  <Import Project="..\..\..\build\Extensions.props" />
-  <ItemGroup>
-    <FrameworkReference Include="Microsoft.AspNetCore.App" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\..\..\src\DotNetWorker\DotNetWorker.csproj" />
-    <ProjectReference Include="..\..\Worker.Extensions.Abstractions\src\Worker.Extensions.Abstractions.csproj" />
-    <ProjectReference Include="..\..\Worker.Extensions.Http\src\Worker.Extensions.Http.csproj" />
-  </ItemGroup>
+    <PropertyGroup>
+        <AssemblyName>Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore</AssemblyName>
+        <RootNamespace>Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore</RootNamespace>
+        <Description>ASP.NET Core extensions for .NET isolated functions</Description>
+        
+        <!--Version information-->
+        <VersionPrefix>1.0.0</VersionPrefix>
+        <TargetFrameworks>net6.0; netstandard2.0</TargetFrameworks>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0'">
+        <OutputType>Library</OutputType>
+        <Nullable>disable</Nullable>
+        <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+        <IncludeBuildOutput>false</IncludeBuildOutput>
+        <PackageTags>Azure Functions Worker, analyzers</PackageTags>
+        <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);_AddAnalyzersToOutput</TargetsForTfmSpecificContentInPackage>
+    </PropertyGroup>
+
+
+    <Import Project="..\..\..\build\Extensions.props" />
+
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0'">
+        <FrameworkReference Include="Microsoft.AspNetCore.App" />
+    </ItemGroup>
+    
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0'">
+        <ProjectReference Include="..\..\..\src\DotNetWorker\DotNetWorker.csproj" />
+        <ProjectReference Include="..\..\Worker.Extensions.Abstractions\src\Worker.Extensions.Abstractions.csproj" />
+        <ProjectReference Include="..\..\Worker.Extensions.Http\src\Worker.Extensions.Http.csproj" />
+    </ItemGroup>
+    
+    <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0'">
+        <Compile Remove="**\*.cs; *.cs" />
+        <Compile Include="Analyzer\*.cs" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.3">
+            <PrivateAssets>all</PrivateAssets>
+            <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+        </PackageReference>
+        <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.11.0" />
+    </ItemGroup>
```

Comment:
For reference, changes here are similar to [Sdk.Analyzers.csproj](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/sdk/Sdk.Analyzers/Sdk.Analyzers.csproj).